api/meta_protocol_proxy/v1alpha/meta_protocol_proxy.proto

```diff
@@ -151,3 +154,10 @@ message Tracing {
   envoy.config.trace.v3.Tracing.Http provider = 9;
 }
 
+message ProtocolConfig {
+  // The codec which encodes and decodes the application protocol.
```

Comment:
Add a name here.

Reply:
add this ? string application_protocol = 2 [(validate.rules).string = {min_len: 1}];

Reply:
Maybe just name? Because the outer struct is ApplicationProtocol, calling it name should be ok.

Reply:
yes, i add name only